Doppler.Sap.Test/Utils/DateTimeProviderTest.cs

```diff
@@ -0,0 +1,54 @@
+using Doppler.Sap.Utils;
+using System;
+using Xunit;
+
+namespace Doppler.Sap.Test.Utils
+{
+    public class DateTimeProviderTest
+    {
+        [Fact]
+        public void DateTimeProvider_ShouldBeReturnDateInArgentinaTimezone_WhenTimeZoneIsNotNull()
+        {
+            var timeZoneConfigurations = new TimeZoneConfigurations
+            {
+                TimeZone = TimeZoneHelper.GetTimeZoneByOperativeSystem("Argentina Standard Time")
+            };
+            var dateTimeProvider = new DateTimeProvider();
+            var utcDate = dateTimeProvider.UtcNow;
+            var cstZone = TimeZoneInfo.FindSystemTimeZoneById(timeZoneConfigurations.TimeZone);
+
+            var expectedDate = utcDate.AddMinutes(cstZone.BaseUtcOffset.TotalMinutes);
```

Comment:
What do you think about testing predefined dates with predefined expected results? Maybe you can keep this test and add others with predefined values, you can test edge cases and use different _Kind_.

Reply:
good, I will add more unit tests

Reply:
I added more uni test to use predefined values and different kind. Let me know if you want that I add more unit tests.